tools/src/main/resources/create-sql-scripts.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
 # Turns off foreign key cheks to speed up script execution
-cat database/mysql_script_header.sql > upgrade/new-install.sql
+cat database/empty_openmrs_dump_1.11.6.sql > upgrade/new-install.sql
```

Comment:
@slubwama The empty database has to be added after the script header not before, also call it platform-sql just like in the SDK